x-pack/plugins/maps/public/classes/layers/vector_layer/vector_layer.js

```diff
@@ -489,7 +518,7 @@ export class VectorLayer extends AbstractLayer {
         layerName,
         style,
         dynamicStyleProps,
-        registerCancelCallback,
+        registerCancelCallback.bind(null, requestToken),
```

Comment:
Found a bug while working on this PR where style meta did not properly register callback.